### ISSUE TITLE
Stuck in audio capture

### ISSUE THREAD

**Issue by rsmith02ct** [Const-me/Whisper#13]
I was able to add a model to Whisper. On the next page I selected audio capture. However there seems to be no way to browse to a file instead. I hit back and it goes back to the model selection screen. Close exits the program. 

[Edit: figured it out- it is "transcribe file" That sounds like transcribe to file, ie output subtitles. I suggest rewording to "load audio file"]

**Comment by rsmith02ct** [comment]
If there's a way to save preferences so you don't have to change the model and format (SRT, etc.) every time it would speed the workflow here. A further boost would be the ability to batch files. 

Very impressive file and I'm getting extremely fast times vs a CPU implementation of Whisper even on a NVIDIA 1050 GPU!

**Comment by Const-me** [comment]
@rsmith02ct The desktop app should save everything in registry. Are you saying that doesn’t work?

If so, do you run any third-party “security” software? Can you run regedit.exe and check is there the following key present: `HKEY_CURRENT_USER\SOFTWARE\const.me\WhisperDesktop` ?

About batching, you can compile “main” or “TranscribeCS” examples. These examples building command-line apps with CLI very similar to the original whisper.cpp. Run them like that:
```
main.exe -m D:\Data\Whisper\ggml-medium.bin -f jfk.wav -otxt
```

**Comment by rsmith02ct** [comment]
Hi Konstantin,
I closed out and reloaded the program and it does seem to retain the model
now as well as the settings, so perhaps that was just an issue on first
usage or otherwise user error.

I looked in RegEdit and see all those values are recorded there. So at this
point it all works!

Appreciate the guidance on batching.

Keep up the great work and shared this software with friends in the video
editing world.
Roger
